unity-renderer/Assets/UIComponents/Scripts/Components/BaseComponentView.cs

```diff
@@ -81,14 +81,13 @@ public interface IComponentModelConfig<T> where T: BaseComponentModel
 
 public abstract class BaseComponentView : MonoBehaviour, IBaseComponentView
 {
-    public ShowHideAnimator showHideAnimator;
-
     internal BaseComponentModel baseModel;
     private bool isDestroyed;
 
+    public ShowHideAnimator showHideAnimator;
     public virtual bool isVisible { get; private set; }
     public bool isFocused { get; private set; }
-
+    public bool isViewEnabled { get; set; } = true;
```

Comment:
Instead of adding this to all components, can you just implement the fix into the `TooltipComponentView`? WDYT?